XCUITests/FxScreenGraph.swift

```diff
@@ -332,4 +341,10 @@ extension Navigator {
             self.goto(TabTray)
         }
     }
+
+    func changeBrowserView(view: String) {
```

Comment:
I have been working on this, but sorry I am not sure I got what you have in mind...is it to avoid using [this locators](https://github.com/mozilla-mobile/firefox-ios/blob/8e44cfbdcca5ed471227ecc0138b45af5f215024/XCUITests/NavigationTest.swift#L165) in the test? 
If so, I got to create an enum:

> enum BrowserMenuChoice: String {
>     case Desktop = "RequestDesktopMenuItem"
>     case Mobile = "RequestMobileMenuItem"
> }

But still need a method to tap on one or the other option in FxScreenGraph similar to changeBrowserView, the difference is how it is called from the test:

> navigator.changeBrowserView(view: BrowserMenuChoice.Desktop.rawValue)

If that is not the purpose of enum I am a bit lost on how to improve it :) , the other options I have tried I do not see how they simplify the current changeBrowserView in FxScreenGraph so that it can be removed in favour of enum without adding calls like goto(BrowserTabMenu) in the test itslef. 
Sorry for asking so much :persevere: @jhugman